starport/templates/app/stargate/readme.md.plush

```diff
@@ -30,6 +28,26 @@ npm run serve
 
 The frontend app is built using the `@starport/vue` and `@starport/vuex` packages. For details, see the [monorepo for Starport front-end development](https://github.com/tendermint/vue).
 
+## Release
+To release a new version of your app, create and push a new tag with `v` prefix. A new draft release with the configured targets will be created.
+
+```
+git tag v0.1
+git push origin v0.1
+```
+
+After a draft release is created, make your final changes from the release page and publish it.
+
+### Install
+To install the latest release, execute following command on your machine:
+
+```
+curl https://get.starport.network/<%= OwnerAndRepoName %>! | sudo bash
+```
+
+**<%= AppName %>** is a blockchain built using Cosmos SDK and Tendermint and created with [Starport](https://github.com/tendermint/starport).
```

Comment:
I think the description should be at the top, though.

Reply:
solved by ef1e1ff.